### ISSUE TITLE
ethermintd dose not support PubkeyMultisigThreshhold

### ISSUE THREAD

**Issue by xiangjianmeng** [cosmos/ethermint#695]
__System info:__ 
1. The anteHandler of cosmos simulation app uses`DefaultSigVerificationGasConsumer` which supports PubkeyMultisigThreshhold. But why the ethermint uses customized `sigGasConsumer` which does not support PubkeyMultisigThreshhold.

![image](https://user-images.githubusercontent.com/18412271/104150529-0d5a4c00-5415-11eb-921c-86da6969ee17.png)

![image](https://user-images.githubusercontent.com/18412271/104150197-139bf880-5414-11eb-89cd-74b745e9c1f6.png)

__Steps to reproduce:__

1. [First Step]
2. [Second Step]
3. [and so on...]

__Expected behavior:__ [What you expected to happen]

__Actual behavior:__ [What actually happened]

__Additional info:__ [Include gist of relevant config, logs, etc.]

**Comment by fedekunze** [comment]
Correct, it doesn't support the default multisigs from the SDK as they are composed of `secp256k1` keys which are not supported by the EVM module. There are 2 main options for how to address this:

1. Add support for this key type and the default SDK multisig but that would mean that a multisig couldn't be used on for EVM module
2. Write an ADR that proposes a custom multisig for `eth_secp256k1` keys and implement it
